### PR TITLE
Add Application Insights to the project templates

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: src/BaseTemplates/StarterWeb/StarterWeb.csproj [hunk 1]
```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="__AspNetCorePatchPackageVersion__" />
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="__AspNetCorePatchPackageVersion__" />
     <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="1.0.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0-beta1" />
```
Comment (phenning) [inline src/BaseTemplates/StarterWeb/StarterWeb.csproj]: Alphabetize

File: src/BaseTemplates/EmptyWeb/EmptyWeb.csproj [hunk 2]
```diff
@@ -18,6 +18,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore" Version="__NetCorePlatformVersion__" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0-beta1" />
```
Comment (phenning) [inline src/BaseTemplates/EmptyWeb/EmptyWeb.csproj]: Alphabetize

File: src/BaseTemplates/WebAPI/WebAPI.csproj [hunk 3]
```diff
@@ -24,6 +24,7 @@
     <PackageReference Include="Microsoft.AspNetCore" Version="__NetCorePlatformVersion__" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="__AspNetCoreMvcPatchPackageVersion__" />
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="__AspNetCorePatchPackageVersion__" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0-beta1" />
```
Comment (phenning) [inline src/BaseTemplates/WebAPI/WebAPI.csproj]: Alphabetize

**Review by phenning — CHANGES_REQUESTED** [review]

File: src/BaseTemplates/StarterWeb/StarterWeb.csproj [hunk 4]
```diff
@@ -21,6 +21,7 @@
   </ItemGroup>$endif$
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0-beta1" />
```
Comment (mlorbetske) [inline src/BaseTemplates/StarterWeb/StarterWeb.csproj]: This version will need to be parameterized, right @phenning?

File: src/BaseTemplates/WebAPI/WebAPI.csproj [hunk 5]
```diff
@@ -21,6 +21,7 @@
   </ItemGroup>$endif$
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0-beta1" />
```
Comment (mlorbetske) [inline src/BaseTemplates/WebAPI/WebAPI.csproj]: This one too

**Review by mlorbetske — APPROVED** [review]

**Pull request merged**